From 99d93bd00d98ee988f6dc60f43be443e01c87ecc Mon Sep 17 00:00:00 2001
From: Dan Villiom Podlaski Christiansen <dchristiansen@zimbra.com>
Date: Tue, 20 Jan 2015 14:13:33 +0100
Subject: [PATCH] The aria-describedby attribute should point to Window titles.

---
diff -aur -x node_modules /tmp/root/tinymce-4.9.11/src/ui/main/ts/Window.ts tinymce-4.9.11/src/ui/main/ts/Window.ts
--- /tmp/root/tinymce-4.9.11/src/ui/main/ts/Window.ts	2022-08-10 05:50:51.895002985 +0000
+++ tinymce-4.9.11/src/ui/main/ts/Window.ts	2022-08-10 05:49:58.097976738 +0000
@@ -188,7 +188,7 @@
       }
     });
 
-    self.aria('describedby', self.describedBy || self._id + '-none');
+    self.aria('describedby', self.describedBy || self._id + '-title');
     self.aria('label', settings.title);
     self._fullscreen = false;
   },
