From 99d93bd00d98ee988f6dc60f43be443e01c87ecc Mon Sep 17 00:00:00 2001
From: Dan Villiom Podlaski Christiansen <dchristiansen@zimbra.com>
Date: Tue, 20 Jan 2015 14:13:33 +0100
Subject: [PATCH] The aria-describedby attribute should point to Window titles.

---
diff -aur -x node_modules /tmp/zimbra/tinymce-4.7.9/src/ui/main/ts/Window.ts tinymce-4.7.9/src/ui/main/ts/Window.ts
--- /tmp/zimbra/tinymce-4.7.9/src/ui/main/ts/Window.ts	2018-03-12 05:42:02.004290037 -0400
+++ tinymce-4.7.9/src/ui/main/ts/Window.ts	2018-03-12 05:38:34.779819234 -0400
@@ -184,7 +184,7 @@
       self.close();
     });
 
-    self.aria('describedby', self.describedBy || self._id + '-none');
+    self.aria('describedby', self.describedBy || self._id + '-title');
     self.aria('label', settings.title);
     self._fullscreen = false;
   },
