diff -aur -x node_modules /tmp/root/tinymce-4.9.11/src/plugins/paste/main/ts/core/Clipboard.ts tinymce-4.9.11/src/plugins/paste/main/ts/core/Clipboard.ts
--- /tmp/root/tinymce-4.9.11/src/plugins/paste/main/ts/core/Clipboard.ts	2020-07-13 04:14:19.000000000 +0000
+++ tinymce-4.9.11/src/plugins/paste/main/ts/core/Clipboard.ts	2022-08-10 06:17:11.765248972 +0000
@@ -375,7 +375,9 @@
     }
 
     // Not a keyboard paste prevent default paste and try to grab the clipboard contents using different APIs
-    if (!isKeyBoardPaste) {
+    // Don't prevent the default if there is no clipboard content (See Bug 97552), and its not IE (which
+    // will crash on trying the default).
+    if (!isKeyBoardPaste && (Env.ie || (Object.keys(clipboardContent).length !== 0))) {
       e.preventDefault();
     }
 
