diff -aur -x node_modules /tmp/root/tinymce-4.9.11/src/themes/modern/main/ts/ui/Toolbar.ts tinymce-4.9.11/src/themes/modern/main/ts/ui/Toolbar.ts
--- /tmp/root/tinymce-4.9.11/src/themes/modern/main/ts/ui/Toolbar.ts	2020-07-13 04:14:19.000000000 +0000
+++ tinymce-4.9.11/src/themes/modern/main/ts/ui/Toolbar.ts	2022-08-10 06:19:45.884202736 +0000
@@ -101,7 +101,8 @@
       classes: 'toolbar-grp',
       ariaRoot: true,
       ariaRemember: true,
-      items: toolbars
+      items: toolbars,
+      ignoreTab: true
     };
   }
 };
diff -aur -x node_modules /tmp/root/tinymce-4.9.11/src/ui/main/ts/KeyboardNavigation.ts tinymce-4.9.11/src/ui/main/ts/KeyboardNavigation.ts
--- /tmp/root/tinymce-4.9.11/src/ui/main/ts/KeyboardNavigation.ts	2022-08-10 06:21:18.757981758 +0000
+++ tinymce-4.9.11/src/ui/main/ts/KeyboardNavigation.ts	2022-08-10 06:20:35.253148504 +0000
@@ -330,7 +330,10 @@
 
       return true;
     } else {
-      return moveFocus(e.shiftKey ? -1 : 1);
+      const navigationRoot = getNavigationRoot(),
+        ignoreTab = navigationRoot && navigationRoot.settings.ignoreTab;
+
+      return !ignoreTab ? moveFocus(e.shiftKey ? -1 : 1) : false;
     }
   }
 
