diff -aur -x node_modules /var/folders/sy/trjn_sh97wq_4pr5f1x7h2k40000gq/T//tinymce-4.1.6/js/tinymce/classes/ui/KeyboardNavigation.js tinymce-4.1.6/js/tinymce/classes/ui/KeyboardNavigation.js
--- /var/folders/sy/trjn_sh97wq_4pr5f1x7h2k40000gq/T//tinymce-4.1.6/js/tinymce/classes/ui/KeyboardNavigation.js	2015-03-13 11:49:18.000000000 -0700
+++ tinymce-4.1.6/js/tinymce/classes/ui/KeyboardNavigation.js	2015-03-13 11:46:17.000000000 -0700
@@ -322,7 +322,10 @@
 
 				return true;
 			} else {
-				return moveFocus(e.shiftKey ? -1 : 1);
+				var navigationRoot = getNavigationRoot(),
+					ignoreTab = navigationRoot && navigationRoot.settings.ignoreTab;
+
+				return !ignoreTab ? moveFocus(e.shiftKey ? -1 : 1) : false;
 			}
 		}
 
@@ -414,4 +417,4 @@
 			focusFirst: focusFirst
 		};
 	};
-});
\ No newline at end of file
+});
diff -aur -x node_modules /var/folders/sy/trjn_sh97wq_4pr5f1x7h2k40000gq/T//tinymce-4.1.6/js/tinymce/themes/modern/theme.js tinymce-4.1.6/js/tinymce/themes/modern/theme.js
--- /var/folders/sy/trjn_sh97wq_4pr5f1x7h2k40000gq/T//tinymce-4.1.6/js/tinymce/themes/modern/theme.js	2014-10-08 02:01:50.000000000 -0700
+++ tinymce-4.1.6/js/tinymce/themes/modern/theme.js	2015-03-13 11:47:13.000000000 -0700
@@ -175,7 +175,8 @@
 				classes: "toolbar-grp",
 				ariaRoot: true,
 				ariaRemember: true,
-				items: toolbars
+				items: toolbars,
+				ignoreTab: true
 			};
 		}
 	}
Only in tinymce-4.1.6/js/tinymce/themes/modern: theme.min.js
