diff -aur -x node_modules /var/folders/vr/zh_nryk119x687x7ksc_7_5m0000gs/T//tinymce-4.9.11/src/plugins/charmap/main/ts/ui/Dialog.ts tinymce-4.9.11/src/plugins/charmap/main/ts/ui/Dialog.ts
--- /var/folders/vr/zh_nryk119x687x7ksc_7_5m0000gs/T//tinymce-4.9.11/src/plugins/charmap/main/ts/ui/Dialog.ts	2020-07-13 09:44:19.000000000 +0530
+++ tinymce-4.9.11/src/plugins/charmap/main/ts/ui/Dialog.ts	2023-01-20 13:12:26.000000000 +0530
@@ -5,6 +5,7 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
+import { HTMLElement } from '@ephox/dom-globals';
 import Actions from '../core/Actions';
 import CharMap from '../core/CharMap';
 import GridHtml from './GridHtml';
@@ -22,6 +23,18 @@
 const open = function (editor) {
   let win;
 
+  const handleMouseOver = function (e) {
+    const td = getParentTd(e.target);
+
+    if (td && td.firstChild) {
+      win.find('#preview').text(td.firstChild.firstChild.data);
+      win.find('#previewTitle').text(td.title);
+    } else {
+      win.find('#preview').text(' ');
+      win.find('#previewTitle').text(' ');
+    }
+  };
+
   const charMapPanel = {
     type: 'container',
     html: GridHtml.getHtml(CharMap.getCharMap(editor)),
@@ -44,17 +57,31 @@
         }
       }
     },
-    onmouseover (e) {
-      const td = getParentTd(e.target);
 
-      if (td && td.firstChild) {
-        win.find('#preview').text(td.firstChild.firstChild.data);
-        win.find('#previewTitle').text(td.title);
-      } else {
-        win.find('#preview').text(' ');
-        win.find('#previewTitle').text(' ');
+    onkeyDown(e) {
+      const chars = Array.from(this.$el.context.querySelectorAll('div'));
+      const totalChars = chars.length;
+      const currentCharIndex = chars.indexOf(e.target);
+      let nextCharIndex = null;
+
+      if (e.keyCode === 37 || e.keyCode === 39 || e.keyCode === 9) {
+        nextCharIndex = currentCharIndex;
+      } else if (e.keyCode === 38 || e.keyCode === 40) {
+        nextCharIndex =
+          e.keyCode === 38
+            ? totalChars + currentCharIndex - 24
+            : currentCharIndex + 24;
       }
-    }
+
+      if (nextCharIndex) {
+        const nextChar = chars[nextCharIndex % totalChars];
+        e.target.blur();
+        (nextChar as HTMLElement).focus();
+        handleMouseOver(e);
+      }
+    },
+
+    onmouseover: handleMouseOver,
   };
 
   win = editor.windowManager.open({
diff -aur -x node_modules /var/folders/vr/zh_nryk119x687x7ksc_7_5m0000gs/T//tinymce-4.9.11/src/skins/lightgray/main/less/desktop/TinyMCE.less tinymce-4.9.11/src/skins/lightgray/main/less/desktop/TinyMCE.less
--- /var/folders/vr/zh_nryk119x687x7ksc_7_5m0000gs/T//tinymce-4.9.11/src/skins/lightgray/main/less/desktop/TinyMCE.less	2020-07-13 09:44:19.000000000 +0530
+++ tinymce-4.9.11/src/skins/lightgray/main/less/desktop/TinyMCE.less	2023-01-17 17:11:07.000000000 +0530
@@ -75,11 +75,17 @@
   line-height: 20px;
   text-align: center;
   vertical-align: middle;
-  padding: 2px;
 }
 
 .@{prefix}-charmap td div {
   text-align: center;
+  padding: 2px;
+  border: 1px solid transparent; 
+}
+
+.@{prefix}-charmap td div:focus-visible,
+.@{prefix}-charmap td:hover div { 
+  border-color: #2275d1; 
 }
 
 .@{prefix}-charmap td:hover {
