Only in tinymce-4.9.11: package-lock.json
diff -aur -x node_modules /var/folders/xt/69w9r4s15573zdgq7588h_t80000gq/T//tinymce-4.9.11/src/plugins/code/main/ts/ui/Dialog.ts tinymce-4.9.11/src/plugins/code/main/ts/ui/Dialog.ts
--- /var/folders/xt/69w9r4s15573zdgq7588h_t80000gq/T//tinymce-4.9.11/src/plugins/code/main/ts/ui/Dialog.ts	2020-07-13 09:44:19.000000000 +0530
+++ tinymce-4.9.11/src/plugins/code/main/ts/ui/Dialog.ts	2022-12-16 13:07:39.000000000 +0530
@@ -18,6 +18,7 @@
       type: 'textbox',
       name: 'code',
       multiline: true,
+      ariaLabel: 'Source code',
       minWidth,
       minHeight,
       spellcheck: false,
diff -aur -x node_modules /var/folders/xt/69w9r4s15573zdgq7588h_t80000gq/T//tinymce-4.9.11/src/ui/main/ts/KeyboardNavigation.ts tinymce-4.9.11/src/ui/main/ts/KeyboardNavigation.ts
--- /var/folders/xt/69w9r4s15573zdgq7588h_t80000gq/T//tinymce-4.9.11/src/ui/main/ts/KeyboardNavigation.ts	2022-12-20 12:44:55.000000000 +0530
+++ tinymce-4.9.11/src/ui/main/ts/KeyboardNavigation.ts	2022-12-16 16:00:46.000000000 +0530
@@ -116,7 +116,7 @@
       return true;
     }
 
-    if (/^(button|menuitem|checkbox|tab|menuitemcheckbox|option|gridcell|slider)$/.test(getRole(elm))) {
+    if (/^(button|menuitem|combobox|checkbox|tab|menuitemcheckbox|option|gridcell|slider)$/.test(getRole(elm))) {
       return true;
     }
 
